### ISSUE TITLE
Email are not sent

### ISSUE THREAD

**Issue by smlavoie** [catalyst/moodle-mod_reengagement#20]
Hi Dan,
I have problem with Reengagement plugin. Emails are not sent. At first, I thought something was wrong in activity settings, but when I got a message stating date and time of sent emails in the activity, I knew activity settings were OK.

So I went into different server logs to try to find the source of the problem. I finally found something in the cron log. Here is one entry:
Execute scheduled task: Tâche crom de réengagement (mod_reengagement\task\cron_task)
... started 08:00:01. Current memory use 6.9Mo.
... used 39 dbqueries
... used 0.32810091972351 seconds
Scheduled task failed: Tâche crom de réengagement (mod_reengagement\task\cron_task),Erreur de lecture de la base de données

I removed and reinstalled the plugin. Just to make sure... same kind of entry in the cron log. Can you help me solve the problem?

For your information, we're running Moodle 3.2.1+ (Build: 20170216) Version 2016120501.08 and mysqlnd 5.0.11-dev.

As per your request "enable the global showcrondebugging setting", since i'm not a sysadmin, I do not know how to do that. Could you be more specific, please?

Thanks for your assistance!

**Comment by danmarsden** [comment]
cron debugging setting should be found in admin > server > debugging when you log in as an admin to the Moodle site.

**Comment by smlavoie** [comment]
Hi Dan, 
In our Moodle (3.2), I found a debugging page at admin>Development. I turned debug message to extra and checked display debug messages. But no cron debugging parameter. I made a search for cron in admin, and found no such parameter as cron debugging. I'm sorry.

I saw in admin>server a scheduled tasks page. The reengagement task is displayed but has never been run.

Thanks again for your assistance!
